Directory.Build.props

```diff
@@ -9,7 +9,7 @@
     <DebugSymbols>true</DebugSymbols>
     <!-- Default to all packages generating a corresponding symbol package -->
     <IncludeSymbols>true</IncludeSymbols>
-    <IsShippingPackage>false</IsShippingPackage>
+    <IsShippingPackage Condition="$(IsShippingPackage) == ''">false</IsShippingPackage>
```

Comment:
I don't think this change is necessary, as this should be evaluated before the property is set in the project file.  So the project file value should override the value set here regardless.